### DISCUSSION TITLE
Add a quick-links folder widget

### DISCUSSION THREAD

**Discussion by wattux** [bluecaret/carettab#375] [Ideas and Feature Requests]
Hey there!

I have a bunch of links I'd like to have a quick access too, but without "poluting" my new-tab page. Something like a folder or drop down to select from. Similar to the classic bookmarks bar folders. Was something like this considered in the past? If so, what was the outcome?

**Comment by bluecaret** [comment]
There is complexity in building out the ability to have nested links within the simple settings panel so I opted not to go that direction and keep it as a handful of quick links. However, you can use your browser's bookmark manager to show those bookmarks instead of adding the links to CaretTab. In the settings for quick links, change the type to "browser's bookmarks" and make sure to allow permission with the button that shows up. After that refresh the tab.

![image](https://github.com/bluecaret/carettab/assets/9217636/69652afa-7734-4ac3-9fb1-a797455d257f)

From here if you have any folders in your bookmarks bar, they will work in CaretTab as well.

![image](https://github.com/bluecaret/carettab/assets/9217636/01ccde90-d08e-4367-ba2b-21a4e56f7140)

**Reply by wattux** [reply]
Seems legit, thanks!
